msg
connecting to ella-HP-ZBook-Studio-G5 54191
Recording to 'S3W2_1.bag'.
Less than 5 x 1G of space free on disk with 'S3W2_1.bag.active'.
"Started controllers: joint_state_controller, torso_controller, head_controller, arm_controller, mobile_base_controller"
Controller Spawner: Loaded controllers: arm_velocity_trajectory_controller
Started controllers: imu_sensor_controller
Started controllers: hand_controller
Started controllers: force_torque_sensor_controller
Waiting for move_base action server...
connecting to ella-HP-ZBook-Studio-G5 45569
connecting to ella-HP-ZBook-Studio-G5 45569
connecting to ella-HP-ZBook-Studio-G5 45569
Connected to move_base action server.
"Sending goal 1 to move_base: x=-1.5, y=-4.5"
Navigating to waypoint 1...
"Transitioning to ACTIVE (from WAITING_FOR_GOAL_ACK, goal: /tiago_autonomous_navigation-1-25.128)"
Failed to get a plan.
Clearing both costmaps outside a square (3.00m) large centered on the robot.
Clearing both costmaps outside a square (1.50m) large centered on the robot.
Rotate recovery behavior started.
Less than 5 x 1G of space free on disk with 'S3W2_1.bag.active'.
Failed to get a plan.
Aborting because a valid plan could not be found. Even after executing all recovery behaviors
"Transitioning to WAITING_FOR_RESULT (from ACTIVE, goal: /tiago_autonomous_navigation-1-25.128)"
"Transitioning to DONE (from WAITING_FOR_RESULT, goal: /tiago_autonomous_navigation-1-25.128)"
Failed to reach waypoint 1
"Sending goal 2 to move_base: x=-3.5, y=-7.0"
Navigating to waypoint 2...
"Transitioning to ACTIVE (from WAITING_FOR_GOAL_ACK, goal: /tiago_autonomous_navigation-2-32.485)"
Clearing both costmaps outside a square (3.00m) large centered on the robot.
Clearing both costmaps outside a square (1.50m) large centered on the robot.
Rotate recovery behavior started.
Failed to get a plan.
Aborting because a valid plan could not be found. Even after executing all recovery behaviors
"Transitioning to WAITING_FOR_RESULT (from ACTIVE, goal: /tiago_autonomous_navigation-2-32.485)"
"Transitioning to DONE (from WAITING_FOR_RESULT, goal: /tiago_autonomous_navigation-2-32.485)"
Failed to reach waypoint 2
